### PR TITLE
Node: Gateway Relayer

### PR DESCRIPTION
**Pull request by bruce-riley**
The Gateway Relayer is a guardian module that looks for payload three transfers destined for the `ibc-translator` contract on Gateway/Wormchain and forwards those VAAs to that contract.

To enable the gateway relayer, specify the following guardian config parameters:
` --gatewayRelayerContract, --gatewayRelayerKeyPath, --gatewayRelayerKeyPassPhrase, --wormchainURL`

### CODE REVIEW

File: node/cmd/guardiand/node.go [hunk 1]
```diff
@@ -1003,6 +1011,40 @@ func runNode(cmd *cobra.Command, args []string) {
 		}
 	}
 
+	var gatewayRelayerWormchainConn *wormconn.ClientConn
+	if *gatewayRelayerContract != "" {
+		if *wormchainURL == "" {
```
Comment (panoel) [inline node/cmd/guardiand/node.go:1016]: Do we want to add any of this to the rpcMap[] for the dumpRpcs command?

File: node/pkg/gwrelayer/gwrelayer.go [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+		ctx:             ctx,
+		logger:          logger.With(zap.String("component", "gwrelayer")),
+		contractAddress: contractAddress,
+		wormchainConn:   wormchainConn,
+		env:             env,
+		subChan:         make(chan *vaa.VAA, subChanSize),
+	}
+}
+
+// Run initializes the accountant and starts the watcher runnable.
```
Comment (panoel) [inline node/pkg/gwrelayer/gwrelayer.go:92]: I didn't see it initializing the accountant...

**Review by panoel — DISMISSED** [review]

File: node/cmd/guardiand/node.go (hunk 1 above)
Comment (bruce-riley) [reply node/cmd/guardiand/node.go:1016]: Well, we don't do it for the accountant or wormchain in general, so doesn't seem like we need it for this?

**Review by panoel — APPROVED** [review]

File: node/pkg/gwrelayer/gwrelayer.go [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+}
+
+// submitVAAToContract submits a VAA to the smart contract on wormchain.
+func (gwr *GatewayRelayer) submitVAAToContract(v *vaa.VAA) error {
+	_, err := SubmitVAAToContract(gwr.ctx, gwr.logger, gwr.wormchainConn, gwr.contractAddress, v)
+	if err != nil {
+		submitErrors.Inc()
+		return err
+	}
+	// TODO: Need to check txResp for "VAA already submitted", which should not be an error.
```
Comment (nik-suri) [inline node/pkg/gwrelayer/gwrelayer.go:196]: eta for this todo?

Comment (bruce-riley) [reply node/pkg/gwrelayer/gwrelayer.go:196]: Once I can talk to the real contract and see how it behaves. :grin:

**Review by nik-suri — APPROVED** [review]

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Default
 
-* @evan-gray
+- @evan-gray
```
Comment (evan-gray) [inline .github/CODEOWNERS:3]: pretty sure this has to stay a star. 🙃 maybe prettier thought this was markdown

**Review by evan-gray — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 4 above)
Comment (bruce-riley) [reply .github/CODEOWNERS]: Removed CODEOWNERS from this PR. Will do it in a separate one.

**Review by panoel — APPROVED** [review]

**Review by nik-suri — APPROVED** [review]

**Review by evan-gray — APPROVED** [review]

**Pull request merged**